package.json

```diff
@@ -53,6 +53,10 @@
     "ts-node": "^10.0.0",
     "typescript": "^4.1.3"
   },
+  "resolutions": {
+    "@oclif/command": "1.8.1",
```

Comment:
Did you check to see if we could just get rid of this dependency?
I assume it's coming in through some `devDep` that still hasn't been updated to `v2` stuff?

Reply:
It's coming from cli-ux (haven't looked into why that is though)